terraform/environment/reduced_fees/main.tf

```diff
@@ -195,3 +195,30 @@ resource "aws_cloudwatch_event_target" "cross_account_put" {
   role_arn  = aws_iam_role.cross_account_put.arn
   provider  = aws.region
 }
+
+resource "aws_cloudwatch_event_rule" "receive_update" {
+  name           = "${data.aws_default_tags.current.tags.environment-name}-receive-update-more-evidence-required"
+  description    = "receive more evidence required update events from remote account"
+  event_bus_name = aws_cloudwatch_event_bus.reduced_fees.name
+
+  event_pattern = jsonencode({
+    source      = ["reduced-fees"],
+    detail-type = ["update"]
```

Comment:
Do we need to include `detail` here as well or are the existing two keys sufficient for the pattern to match?

Reply:
This is sufficient for a match. It's what's required to see an event and identify what event bus it should be directed to. 

`detail` will contain some specific as yet undefined content that we should process, such as an item id, and a new status (or anything else the event producer needs to supply).

Reply:
There's information about filtering here https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns-content-based-filtering.html